main.py

```diff
@@ -22,9 +22,11 @@
 procedures = ['Стрижка', 'Гоління', 'Комплекс', 'Написати адміністратору']
 
 
+phone_number = None  #Global variable to store the phone number
+
 @bot.message_handler(commands=['start'])
 def phone(message):
```

Comment:
I think naming is wrong. I woudl like to see something like get_phone_number not phone.